### ISSUE TITLE
pypi v7.1.0 missing  rdbms module in graph component

### ISSUE THREAD

**Issue by obonyojimmy** [neuml/txtai#707]
Looks like v7.1.0 in pypi is missing rdbms module , below related log trace .

```
self.graph = self.creategraph()
taskflow-1  |   File "/root/.local/lib/python3.10/site-packages/txtai/embeddings/base.py", line 986, in creategraph
taskflow-1  |     return GraphFactory.create(config)
taskflow-1  |   File "/root/.local/lib/python3.10/site-packages/txtai/graph/factory.py", line 35, in create
taskflow-1  |     graph = GraphFactory.resolve(backend, config)
taskflow-1  |   File "/root/.local/lib/python3.10/site-packages/txtai/graph/factory.py", line 58, in resolve
taskflow-1  |     raise ImportError(f"Unable to resolve graph backend: '{backend}'") from e
taskflow-1  | ImportError: Unable to resolve graph backend: 'rdbms'
```

**Comment by davidmezzetti** [comment]
Hello, this feature (#699) will be released with v7.2. If you're interested in using it now, you can [install from source](https://neuml.github.io/txtai/install/#install-from-source)

**Comment by obonyojimmy** [comment]
Thanks , that was helpfull
